README.md

```diff
@@ -23,18 +23,30 @@ The above code is all that is needed from the screen starting this flow. Each sc
 
 As you continue to develop your applications, each view controller will become more decoupled from the rest of the app.  That means, if you want a completely different order of screens, just define a new [Workflow](https://wwt.github.io/SwiftCurrent/Classes/Workflow.html).
 
+## See it in action with our sample app
+
+Clone our repo, open `SwiftCurrent.xcworkspace`, target the `SwiftCurrentExample` scheme, and run to see our sample app in action. The app is designed to give you an idea of what SwiftCurrent can do with minimal overhead in the UI.
+
 ## Interested but you need SwiftUI support?
+
 [We're working on it now!](https://github.com/wwt/SwiftCurrent/milestone/2)
 
 # Quick Start
 
-## CocoaPods
-```ruby
-pod 'SwiftCurrent/UIKit'
+This quick start uses SPM, but if you would prefer CocoaPods, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

Comment:
"but if you would prefer CocoaPods". Is this really what we want to say? In the spirit of keeping documentation from getting out of date, "But if you're interested in using a different installation method....".

Reply:
How about?

```suggestion
This quick start uses SPM, but for a complete list of supported package managers, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

Reply:
It's acceptable, but the really pedantic side of me wants to say that our installation instructions *could* include instructions without a package manager. I certainly strongly recommend that people consume our library through a package manager of some kind, but not everyone will choose to do that.

Reply:
Something I tossed around looking at this spot before was

```suggestion
This quick start uses SPM, but for other dependency approaches, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

Reply:
Just dropping in Tyler's suggestion
```suggestion
This quick start uses SPM, but if you're interested in using a different installation method, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```

Reply:
I think I like Morgan's the best so far.  Maybe:

```suggestion
This quick start uses SPM, but for other approaches, [see our installation instructions](https://github.com/wwt/SwiftCurrent/wiki/Installation).
```